Dockerfile

```diff
@@ -1,39 +1,118 @@
-# Build the manager binary
-FROM golang:1.17 as builder
+FROM --platform=$BUILDPLATFORM crazymax/goreleaser-xx:1.2.5 AS goreleaser-xx
```

Comment:
Using goreleaser-xx or goreleaser is overkill here as the goal of this project seems to just release an image and not artifacts.